act-rules-format.bs

```diff
@@ -89,17 +89,40 @@ An ACT Rule MUST have a description that is in plain language and provides a bri
 Accessibility Requirements {#structure-accessibility-requirements}
 ==================================================================
 
-<div class=note>
-  **Editor's note:** The ACT Task Force is looking for feedback about the use of the term "pass" in relation to rules. While rules can "pass", their corresponding accessibility requirement can fail. A section has been added to make this explicit in the rule, but we would like to know if this is sufficient.
+Accessibility requirements are just that: A requirement that a particular web page must conform to for it to be considered accessible. This can (and usually does) include the WCAG success criteria. Often organizations have additional requirements which may come from different sources, such as laws, internal standards, etc. These too are considered accessibility requirements and can be tested using the ACT Rules Format. What the precise requirements are for any test is beyond the scope of the ACT Rules Format.
+
+[=Atomic rules=] and [=composite rules=] SHOULD identify the accessibility requirements that are *not satisfied* when the outcome of a rule is Fail. An ACT Rule is a complete or partial test for one or more accessibility requirements. This means that most ACT rules test only part of an accessibility requirement, but it MUST NOT test more than the accessibility requirement it lists.
+
```

Comment:
Editorial suggestion: "An ACT Rule is a complete or partial test for one or more accessibility requirements but most ACT Rules test only part of an accessibility requirement. However, an ACT Rule MUST NOT test more than the accessibility requirement it lists." (to me the current text is confusing because it mixes different thoughts without clear separation between them)